content/docs/command-reference/stage/list.md

```diff
@@ -67,9 +67,9 @@ DVC also uses this command to provide autocompletion in `bash` and `zsh`.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Example: List stages
+## Examples:
 
-Listing stages from the default `./dvc.yaml` file from the repo.
+The base case is to list stages from `./dvc.yaml`:
```

Comment:
Seems like there should be integration between this section and the `targets` option. At least, they should be part of a common example repo structure with the same `dvc.yaml` files, directories, and stage names. Maybe one should also reference the other so it's clear where to see more in-depth examples?

Reply:
Would this be a more appropriate place to include the `foreach` content (https://github.com/iterative/dvc.org/pull/2245#discussion_r614368197)?

Reply:
> integration between this section and the targets option

OK context: Actually `targets` is not an option (it's the one required argument) and I was kind of opposed originally to put in under **Options**, but @skshetry and @shcheklein beat my vote. They were right in that it's gotten so complex that it needs to be explained in detail but we also didn't want to introduce an **Arguments** section just for that.

Your idea of an example (hypothetical I assume) repo structure to demo al the kinds of targets is great but that would be even longer that the current bullet list so it'd have to be a new/improved Example which I don't think could replace the `targets` bullet list ref.

Reply:
> Would this be a more appropriate place to include the foreach content?

This PR is a follow up to that one actually. I removed the `foreach` terminology here as you suggested. Maybe I'm not getting the Q? 😋

Reply:
> integration between this section and the targets option

Another idea is to move the main argument ino the Synopsis section but that would make the intro a bit long. Or make a **Targets** section in the description but it would have to be as compressed as the current bullet list as the Desc is already long.

Reply:
> OK context: Actually targets is not an option (it's the one required argument) and I was kind of opposed originally to put in under Options, but @skshetry and @shcheklein beat my vote. 

yep, we didn't want to overcomplicate this and fixing two things at a time (potentially needed docs refactoring to accommodate options and arguments) and actually documenting this command. I think it's not a major priority and it's better to first discuss it in the ticket, then apply to all occurrences. WDYT?

Reply:
I agree, not in the scope of this PR. I'm sure it will come up again if needed.